doc/library-maintainer.rst

```diff
@@ -139,6 +139,9 @@ How to Make a Release
    * For minor and major releases, the full release notes text for the
      release, including descriptions of migration instructions, deprecations,
      new features, etc.
+   * If patch releases contains any release notes, those should be copied too.
```

Comment:
I'd suggest melding this with the above:
"
For patch releases, links to the issues fixed in the release (or the PRs which fixed the issues), along with either a one-line description of what was fixed (e.g. the PR or issue title) or more detailed release notes providing a more detailed explanation about the importance of the fix and its potential consequences.
"

Reply:
Actually, maybe this is better:
"
For patch releases, links to the issues fixed in the release (or the PRs which fixed the issues), along with the full release notes text including detailed explanations about the importance of the fixes and their potential consequences. (If a patch release does not have pre-written release notes, the PR or issue title may be included.)
"